crates/syntax/src/ast/token_ext.rs

```diff
@@ -616,3 +612,40 @@ impl Radix {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::ast::{make, FloatNumber, IntNumber};
+
+    fn check_float_suffix<'a>(lit: &str, expected: impl Into<Option<&'a str>>) {
+        assert_eq!(FloatNumber { syntax: make::tokens::literal(lit) }.suffix(), expected.into());
+    }
+
+    fn check_int_suffix<'a>(lit: &str, expected: impl Into<Option<&'a str>>) {
```

Comment:
Oh, `impl Into<Option<_>>` is lovely! I haven't realized it can be used for tests like this, thanks!

This doesn't really matter, but it's generally a good idea to annotate `check` functions with `#[track_caller]` attribute now that that is stable. 

bors +

Reply:
bors r+